Week1/homework/create-table.js

```diff
@@ -0,0 +1,25 @@
+const mysql = require('mysql');
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'newWorld'
+});
+
+con.connect();
+
+const create_query = [
+  "create table country (name varchar(50), continent varchar(50), region varchar(50), surfacearea int, indepyear int,population int,lifeexpectancy int,gnp int,gnpold int,localname varchar(50),govermentform varchar(50),headofstate varchar(50),capital varchar(50))",
```

Comment:
looks good to me, if you can indent this line by line, like 
`create table country (
      name varchar(50),
      continent varchar(50),
      ....
`

Would be better for readability. I mean for the next time. Otherwise it is good

Reply:
Thank you for your time and feedback. I will make it better next time as you mentioned, many thanks!